### DISCUSSION TITLE
Tasmota and IKEA VINSTYRKA SEN54

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#19448] [Show and tell]
My notes and photos of adding Tasmota to the IKEA VINSTYRKA

~~~
https://github.com/trlafleur/Tasmota-VINDSTYRKA
~~~

**Comment by barbudor** [comment]
Hi Tom, thanks for sharing
I wonder what is this wireless module already in the device ? 
Is that Wifi or Bluetooth ?
I wonder if there could be any interrest in changing it for an ESP module though

It's 29.99€ here, I may try to get one and look into that path

**Comment by trlafleur** [comment]
The module will support both BlueTooth and Zigbee, but it appears to use
only Zigbee in this device.
The code in the module also drives the LCD controller chip as well as the
SEN54 device.

https://www.silabs.com/wireless/zigbee/efr32mg21-series-2-modules/device.mgm210la22jnf?tab=specs


~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur

**Reply by barbudor** [reply]
Thanks

**Comment by trlafleur** [comment]
After some testing, I've noted some errors in the readings..

Temp: Tasmota = 88.7f, IKEA VINSTYRKA display = 86f, Kestrel 4000 Reference = 84.1f
Hum:  52.6%, 60%, 59.6%

**Reply by sfromis** [reply]
Maybe the Tasmota values are what the sensor actually provides, and the standard controller board adjusting for the "suboptimal" placement of the sensor module inside a box with heat-emitting electronics. Higher temperature and lower humidity would be consistent with this.
